code/modules/research/designs/mechfabricator_designs.dm

```diff
@@ -653,6 +653,16 @@
 	construction_time = 200
 	category = list("Exosuit Equipment")
 
+/datum/design/medical_jaw
+	name = "Exosuit Medical Equipment (Rescue Jaw)"
+	id = "mech_medical_jaw"
+	build_type = MECHFAB
+	req_tech = list("biotech" = 4, "engineering" = 4)
```

Comment:
design note: the rescue jaw allows progressing from biotech 4->biotech 5

Reply:
This should honestly be the same cost and tech levels as the jaws of life--same origin tech too.

Reply:
You could argue it would be a bit easier to make a mech-sized tool capable of forcing doors than it is to make it handheld, but alright. Changing.